pyaedt/edb_core/edb_data/components_data.py

```diff
@@ -796,6 +796,20 @@ def placement_layer(self):
         """
         return self.edbcomponent.GetPlacementLayer().Clone().GetName()
 
+    @property
+    def is_top(self):
+        """Check if a component is placed at top layer.
+
+        Returns
+        -------
+        bool
+            ``True`` component is placed at top ``False`` at bottom.
+
+        """
+        if self.placement_layer == list(self._pedb.stackup.signal_layers.values())[0].name:
+            return True
+        return False
```

Comment:
It returns False if the component is place on layer 2. It conflicts with the docstring.

Reply:
right let me fix this